docs/maintenance/backup-restore.md

```diff
@@ -0,0 +1,73 @@
+<!--
+title: "Backing up Netdata"
+description: "Guide on how to backup and restore Netdata."
+custom_edit_url: "https://github.com/netdata/netdata/edit/master/docs/maintenance/backup-restore.md"
+sidebar_label: "Notify"
+learn_status: "Published"
+learn_rel_path: "Integrations/Notify"
+-->
```

Comment:
This is not needed, but it doesn't break anything either, remnants of an older time :smile: 
```suggestion
```